geom/src/tessellation.rs

```diff
@@ -180,9 +205,44 @@ impl Tessellation {
         }
         result
     }
+
+    /// Produces a GeoJSON multipolygon consisting of individual triangles. Optionally map the
+    /// world-space points back to GPS.
+    pub fn to_geojson(&self, gps: Option<&GPSBounds>) -> geojson::Geometry {
+        let mut polygons = Vec::new();
+        for triangle in self.triangles() {
+            let raw_pts = vec![triangle.pt1, triangle.pt2, triangle.pt3, triangle.pt1];
+            let mut pts = Vec::new();
+            if let Some(gps) = gps {
+                for pt in gps.convert_back(&raw_pts) {
+                    pts.push(vec![pt.x(), pt.y()]);
+                }
+            } else {
+                for pt in raw_pts {
+                    pts.push(vec![pt.x(), pt.y()]);
+                }
+            }
+            polygons.push(vec![pts]);
+        }
+
+        geojson::Geometry::new(geojson::Value::MultiPolygon(polygons))
+    }
+
+    // TODO This only makes sense for something vaguely Ring-like
+    fn to_geo(&self) -> geo::Polygon {
+        let exterior = crate::conversions::pts_to_line_string(&self.points);
+        geo::Polygon::new(exterior, Vec::new())
+    }
+
+    // TODO After making to_outline return a real Polygon, get rid of this
```

Comment:
Likewise, this is used in the LTN tool to try and clean up a visual artifact from coloured cells